ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/AuthService.scala

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.participant.state.v1
+
+import java.util.concurrent.CompletionStage
+
+/** An interface for authorizing the ledger API access to a participant.
```

Comment:
```suggestion
/** The interface to implement for authorizing the Ledger API access to a participant.
```

Reply:
I will leave the comment as is, it uses the same style as `ReadService` and `WriteService`.